app/src/main/res/drawable/ic_merge_icon_state.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:drawable="@drawable/ic_merge_icon_24" android:state_enabled="true" />
+  <item android:drawable="@drawable/ic_merge_icon_24_disabled" android:state_enabled="false" />
+</selector>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
done